rllib/core/rl_module/rl_module.py

```diff
@@ -111,6 +113,16 @@ def from_dict(cls, d):
             catalog_class=catalog_class,
         )
 
+    def update(self, other):
```

Comment:
type hint plz.

Reply:
Done. Btw I saw that many methods in this file are missing typhints. After it stabelizes, we should do a review of the whole RLModules project concerning typehints and other easy to spot stuff that we tend to forget.